blog/_src/posts/2024-08-14-racket-v8-14.md

```diff
@@ -1,45 +1,49 @@
     Title: Racket v8.14
     Date: 2024-08-14T13:13:03
     Tags: DRAFT
-    Author: John Clements, Stephen de Gabrielle
+    Author: John Clements, Stephen De Gabrielle
 
 
-*posted by Stephen de Gabrielle*
+*posted by Stephen De Gabrielle*
 
 ----------------------------------------------------------------------
 
+We are pleased to announce Racket v8.14 is now available from [https://download.racket-lang.org/](https://download.racket-lang.org).
+
+## As of this release:
+
 - 64-bit ARM Windows is one of the standard pre-built downloads.
 
 - The `compiler/demod` language integrates demodularization into the
   program rather than an external tool. Using the language approach
   cooperates with other Racket tools and allows demodularization to be
   used on libraries. Additionally, demodularization handles submodules
   and can prune definitions, significantly shrinking the size of some
-  programs.
+  programs.  [`#lang compiler/demod`](https://docs.racket-lang.org/raco/demod.html#%28mod-path._compiler%2Fdemod%29)
```

Comment:
Do we want to add parens around this link too?

Reply:
I'm going to try rendering, and then make any needed fixes. Thank you so much for your work on this.